samples/GalleryApp/Models/Skill.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GalleryApp
+{
+    public class Skill
```

Comment:
>Skill [](start = 17, length = 5)

nit: I find the use of the name "Skill" a bit confusing alongside the Microsoft.AI.Skills.x classes (ISkill, etc.) eventhough it is defined in a different namespace. Could be named differently to alleviate this (i.e. SkillGalleryEntry, SkillView, other?). It may be only me though :\